Intersect.Server/Entities/Player.cs

```diff
@@ -1359,16 +1359,16 @@ public void WarpToSpawn(bool sendWarp = false)
         /// </summary>
         /// <param name="item"></param>
         /// <returns></returns>
-        public bool CanGiveItem(Item item)
+        public bool CanGiveItem(Guid itemId, int quantity)
```

Comment:
Need to update the doc if you're changing the method signature, but can you just include both instead of just this one? `Item` can have extra data on it which _may_ be desirable to be used in this method in the future.

Reply:
Ah, I forgot to change this back. I was experimenting with making all methods use id and quantity, since thhe take methods use that. But then realised there was a reason GiveItem used the Item class.. Since items might have specific data fields filled that would need to transfer over.

I'm changing the main method back and adding an overload now.